source/default/_meta/_00-head.twig

```diff
@@ -13,7 +13,7 @@
 		<!-- End Pattern Lab -->
 
     {% if env == 'production' %}
-    <link rel="stylesheet" href="{{ paths.assets }}/app-pl.styles.css?cacheBuster={{ randomNumber }}">
+    <link rel="stylesheet" href="{{ paths.assets }}/app.styles.css?cacheBuster={{ randomNumber }}">
```

Comment:
Not necessarily tied to this PR, but rather than using Faker to generate a random number, would it be better to use Twig's built in Random function? e.g., 
```
<link rel="stylesheet" href="{{ paths.assets }}/app.styles.css?cacheBuster={{ random(10000000) }}">
```

Reply:
@jryanconklin this is a perfect thing I would have never caught. Can you open a quick issue for it?

Reply:
On it!

Reply:
Oh nice! Looks like you beat me to it.